pkg/doc/verifiable/linked_data_proof_test.go

```diff
@@ -41,7 +44,58 @@ func Test_keyResolverAdapter_Resolve(t *testing.T) {
 		kra := &keyResolverAdapter{pubKeyFetcher: SingleKey(privateKey.Public())}
 		resolvedPubKey, err := kra.Resolve("any")
 		require.Error(t, err)
-		require.EqualError(t, err, "expecting []byte public key, got something else")
+		require.EqualError(t, err, "expecting []byte public key")
 		require.Nil(t, resolvedPubKey)
 	})
 }
+
+type dummyKeyResolver struct {
+	pubKey []byte
+}
+
+func (kr *dummyKeyResolver) Resolve(string) ([]byte, error) {
+	return kr.pubKey, nil
+}
+
+// This example is generated using https://transmute-industries.github.io/vc-greeting-card
+func TestLinkedDataProofVerifier(t *testing.T) {
+	pubKeyStr := base58.Decode("5fYcSLLNTzCzCRTP7dNJiQ4AcUqB9Jf63Y4bPJ6abEFe")
+	pubKey := ed25519.PublicKey(pubKeyStr)
+
+	//nolint:lll
+	vcStr := `
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://transmute-industries.github.io/vc-greeting-card/context/vc-greeting-card-v0.0.jsonld"
```

Comment:
Will the test break if this URL goes away?

Reply:
Yes...

Reply:
Fixed!